include/nncase/runtime/stackvm/opcode.h

```diff
@@ -146,6 +146,7 @@ enum class tensor_function_t
     TAKE = 0x0015,
     TRANSPOSE = 0x0016,
     UNARY = 0x0017,
+    MIRROR_PAD = 0x0018,
```

Comment:
You just add another pad mode, no additional stackvm opcode is needed.

Reply:
Reduced it.